src/store/houses/actions.ts

```diff
@@ -12,4 +13,29 @@ export const actions: ActionTree<HousesStateInterface, RootStateInterface> & Act
   [HousesActionsEnum.SET_MODAL_DISPLAY]: ({ commit }, payload) => {
     commit(HousesMutationsEnum.SET_MODAL_DISPLAY, payload);
   },
+  [HousesActionsEnum.EDIT_HOUSE]: async ({ commit }, payload) => {
+    try {
+      const payloadtoSend: HouseDTOModel = {
+        quantity_flat: payload.quantity_flat,
+        house_area: payload.house_area,
+        adjoining_area: payload.adjoining_area,
+        address: payload.address,
+      };
+      await HTTP.put(`/cooperations/1/houses/${payload.id}`, payloadtoSend).then((r) => {
+        console.log(r.data);
+        commit(HousesMutationsEnum.EDIT_HOUSE, r.data);
+      });
+    } catch (err: any) {
+      console.log('error EDIT_HOUSE', err);
+    }
+  },
+  [HousesActionsEnum.DELETE_HOUSE]: async ({ commit }, payload) => {
+    try {
+      await HTTP.delete(`/cooperations/1/houses/${payload.id}`).then(() => {
```

Comment:
And here?

Reply:
fixed

Reply:
Not fixed - hardcoded ID is still there)

Reply:
removed hardcoded cooperation ID